payu/payu.php

```diff
@@ -1,156 +1,229 @@
 <?php
 /**
- *  @copyright  Copyright (c) 2009-2013 TechJoomla. All rights reserved.
- *  @license    GNU General Public License version 2, or later
+ * @copyright  Copyright (c) 2009-2013 TechJoomla. All rights reserved.
```

Comment:
Thank you. Updated!!

On Thu, Mar 24, 2016 at 3:46 PM, Manoj Londhe <notifications@github.com>
wrote:

> In payu/payu.php
> <https://github.com/techjoomla/joomla-payments/pull/13#discussion_r57296919>
> :
>
> > @@ -1,156 +1,229 @@
> >  <?php
> >  /**
> > - *  @copyright  Copyright (c) 2009-2013 TechJoomla. All rights reserved.
> > - *  @license    GNU General Public License version 2, or later
> > + * @copyright  Copyright (c) 2009-2013 TechJoomla. All rights reserved.
>
> @vidyasagarmundhe <https://github.com/vidyasagarmundhe> Update copyright,
> and year.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/techjoomla/joomla-payments/pull/13/files/b1cbedd7ffe01bdde0ccafa425476281184c81dd#r57296919>
>



-- 
Warm Regards,
Vidyasagar Mundhe
*Developer at Techjoomla | Tekdi Technologies Pvt. Ltd.*
+91 20 6050 2226 | +91 8275055441
Web     : www.tekdi.net | www.techjoomla.com
Twitter : @vbmundhe
Blog    : http://vbmundhe.blogspot.com